sites/website/src/docs/introduction.md

```diff
@@ -4,3 +4,39 @@ title: Introduction
 sidebar_label: Introduction
 custom_edit_url: https://github.com/microsoft/fast-dna/edit/master/sites/website/src/docs/introduction.md
 ---
+
+Welcome to the FAST-DNA documentation! We're glad you're here. We can't wait to show you around.
+
+## What is FAST-DNA?
```

Comment:
I think although the project is FAST-DNA we've been largely referring to it in documentation etc. as just FAST. @chrisdholt for opinion.

Reply:
@chrisdholt, yeah I wasn't sure when we were going to make the switch in terminology either.